### ISSUE TITLE
Cannot access block toolbar in 'Top toolbar' mode

### ISSUE THREAD

**Issue by enriquesanchez** [WordPress/gutenberg#25841]
**Describe the bug**
When 'Top toolbar' is enabled, I'm unable to access the block's toolbar with my keyboard. Normally, I'd expect that pressing <kbd>Shift</kbd> + <kbd>Tab</kbd> will move my focus to the block toolbar.

**To reproduce**
1. Add a block to the editor
2. Press <kbd>Shift</kbd> + <kbd>Tab</kbd> repeatedly
3. Notice how focus never moves to the block toolbar

**Expected behavior**
Pressing <kbd>Shift</kbd> + <kbd>Tab</kbd> should move my focus to the block toolbar.

**Screenshots**
![2020-10-05 12-17-36 2020-10-05 12_19_26](https://user-images.githubusercontent.com/3276087/95111692-79862f00-0705-11eb-95c6-c57e02f16614.gif)

**Editor version (please complete the following information):** 
- WordPress version: 
**5.5.1**
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default?
**Gutenberg plugin**
- If the Gutenberg plugin is installed, which version is it? 
**9.1.0**

**Desktop (please complete the following information):**
 - OS: MacOS 10.15.6
 - Browser: Firefox 80.0.1 and Safari 14.0
